src/viam/sdk/services/mlmodel/mlmodel.hpp

```diff
@@ -47,7 +47,7 @@ class MLModelServiceRegistration : public ResourceRegistration {
 /// This class acts as an abstract base class to be used by any driver
 /// implementing an MLModel. It is also used as the base class for the
 /// gRPC client for communicating with a remote MLModel instance.
-class MLModelService : public Service {
+class MLModelService : public Service, public Stoppable {
```

Comment:
See my other comment about `stop`ping the service with `StopAll` / when a modular `MlModelService` is removed or reconfigured.

Reply:
Per discussion, `MLModelService` should not be `Stoppable` because `Stoppable` is for `Resources` which have a GRPC `Stop` entry point, and `MLModelService` does not. Let's remove this and see what the fallout looks like for `MLModelServiceTFLite`, which may offer some clues as to what should be done with its `stop` method.

Reply:
`MLModelService` is no longer stoppable, and I've renamed `MLModelServiceTFLite` to `close` to avoid confusion. Let me know what you think.